### ISSUE TITLE
Portable mode for Ledger Live Desktop App

### ISSUE THREAD

**Issue by beppe9000** [LedgerHQ/ledger-live-desktop#3769]
I would like to put the app in a self-contained usb key and avoid leaving any trace on PCs I use.

**Comment by GentlemansKiller** [comment]
Yes!! I really hope this is implemented. On a side note, you can do this with the Linux AppImage, but obviously not windows :/

**Comment by declancockburn** [comment]
Agreed. Mainly having my business laptop without admin-rights this would be a very useful feature.

**Comment by mariozidov** [comment]
Very useful... especially now with MM problem. 
I can't use Metamask with Ledger anymore

**Comment by beppe9000** [comment]
Me neither.... I've never been able to.
